src/handlers/s3.js

```diff
@@ -1,15 +1,50 @@
 const { AwsClient } = require('aws4fetch');
 const utils = require('../utils');
 
-function s3HandlerFactory({ accessKeyId, secretAccessKey, bucket, region }) {
+function getEndpoint(endpoint, options) {
```

Comment:
I've been having issues with code missing the last couple of weeks when doing PR's between branches. This code was merged in the S3-pr right?

Reply:
Update the branch now and it looks fine!